### DISCUSSION TITLE
MCP GitHub switching branches

### DISCUSSION THREAD

**Discussion by rdtiv** [modelcontextprotocol/.github#71] [Q&A]
### Pre-submission Checklist

- [X] I have checked that this question would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions and documentation for answers

### Question Category

- [ ] Protocol Specification
- [ ] SDK Usage
- [ ] Server Implementation
- [X] General Implementation
- [ ] Documentation
- [ ] Other

### Your Question

I apologize in advance if this is known issue, and I could not find.

I cannot get Claude to switch branches in my repository.

Thanks for helping clarify for a newbie.

**Comment by jspahrsummers** [comment]
Per [the README](https://github.com/modelcontextprotocol/servers/blob/06d4f06072f370e163834bcbc011d169ce9846e9/src/git/README.md), this is not a feature currently supported by the Git MCP server (I assume this is what you meant, rather than GitHub). https://github.com/modelcontextprotocol/servers/pull/166 is an open PR proposing to add this functionality.
